### PR TITLE
Disable interactivity in messaging and Rx cards

### PR DESCRIPTION
**Pull request by kfrz**
## Description
Removed interactive elements from the MessagingWidget and PrescriptionCard components, as they are linking to MHV in an unexpected manner, and this work is out of scope for WBC. See [#14499](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14499) and [#13839](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13839) for more context, as well as [this DSVA slack thread](https://dsva.slack.com/archives/C909ZG2BB/p1540314756000100)

## Testing done
Manual testing

## Screenshots
screencast-localhost-3001-2018 10 24-07-18-06

## Acceptance criteria
- [x] Remove all clickable actions (including hiding buttons) on the prescriptions health widget other than leaving the "View all.." link as is
- [x] Remove all clickable actions on the secure messages health widget other than leaving the "View all.." link as is

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by BBarone1** [comment]
copying comment from slack for visibility:

This is fantastic. The only other piece we'd want here is to remove / hide those yellow highlighting boxes on messages so the user doesn't get confused and think they can click.

we'll want to restore those highlighting boxes and the buttons on Rx in the future, so we want to keep this flexible to reactivate at that time

**Comment by kfrz** [comment]
cc: @1Copenut 

Do we need to do anything more specific here to keep 508-compliant? We're still rendering `<Link>` components for the messaging entries, but we want the link to be completely disabled, thus removing `href` attribute. The elements are no longer in the navigation tab order but content is in-flow. Thanks for the :eyes:

File: src/applications/personalization/dashboard/containers/MessagingWidget.jsx [hunk 1]
```diff
@@ -38,8 +38,12 @@ class MessagingWidget extends React.Component {
       { label: 'Date', value: 'sentDate', nonSortable: true },
     ];
 
+    // eslint-disable-next-line
```
Comment (erikphansen) [inline src/applications/personalization/dashboard/containers/MessagingWidget.jsx]: Why is this needed? To silent the warning about the unused `id` param?

**Review by erikphansen — COMMENTED** [review]
Is this functionality going to be restored soon?

File: src/applications/personalization/dashboard/containers/MessagingWidget.jsx (hunk 1 above)
Comment (kfrz) [reply src/applications/personalization/dashboard/containers/MessagingWidget.jsx]: > Why is this needed? To silent the warning about the unused id param?

Yep


> Is this functionality going to be restored soon?
Functionality will be restored post WBC - we have work scoped out [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13839) and other tickets.

Comment (erikphansen) [reply src/applications/personalization/dashboard/containers/MessagingWidget.jsx]: thanks! As long as that `eslint-disable` is removed when functionality is restored and it's no longer needed it's cool with me.

**Review by erikphansen — APPROVED** [review]

**Review by 1Copenut — APPROVED** [review]
I've reviewed the changes in `localhost` using VoiceOver. They didn't cause any errors with VO, and there were no axe-core errors.

I cannot check these using JAWS or NVDA yet, because of CORS boundaries on serving the site locally via IP. Please ping me when we're in Staging, and I will review with other browsers.

I'm leaving one comment about adding a CSS rule so the secure message links without `href` attributes do not show the click hand, but rather the default arrow. I'd recommend adding it for visual understanding if we can ensure the rule is silo'ed to this one view.

**Review by bshyong — APPROVED** [review]
For the Link, you can also replace with `<a></a>` as an alternative
